app/assets/stylesheets/custom.css.scss

```diff
@@ -187,6 +187,15 @@ h6 {
   font-size: 100px;
 }
 
+.cart-header {
+  border-bottom: 2px solid #dddddd;
```

Comment:
Color literals like `#dddddd` should only be used in variable declarations; they should be referred to via variable everywhere else.